protocol_rfcs/README.md

```diff
@@ -0,0 +1,63 @@
+# Protocol RFCs
+
+This directory contains information about the process of making Delta protocol changes via RFCs and all the RFCs that have been proposed since
+ this process was adopted.
+
+
+## Table of RFCs
+
+This file contains the list of all the protocol RFCs that have been proposed. The details of each RFC are present in the associated .md file. 
```

Comment:
This wording doesn't quite seem to reflect the three-table approach we're now using? Should we update it?
Also, if the process for adding an RFC is in the same place, we should probably create a ToC of jump links at the top of the file, so that people can easily find it even as the growing RFC list pushes it farther down screen?

Reply:
How about this.
```suggestion
Here is the history of all the RFCs propose/accepted/rejected since Feb 6, 2024, when this process was enforced.
```

Reply:
and added TOC at the top.

Reply:
fair. changed.